jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -626,6 +627,39 @@ boolean isSkipped() {
     return skip;
   }
 
+  /**
+   * Return false if the `jib.containerize` property is specified and does not match this
+   * module/project. Used by the Skaffold-Jib binding.
+   *
+   * @return true if this module should be containerized
+   */
+  boolean isContainerizable() {
+    String moduleSpecification = getProperty(PropertyNames.CONTAINERIZE);
+    if (project == null || Strings.isNullOrEmpty(moduleSpecification)) {
+      return true;
+    }
+    // modules can be specified in one of three ways:
+    // 1) a `groupId:artifactId`
+    // 2) an `:artifactId`
+    // 3) relative path within the repository
+    if (moduleSpecification.equals(project.getGroupId() + ":" + project.getArtifactId())
+        || moduleSpecification.equals(":" + project.getArtifactId())) {
+      return true;
+    }
+    // Relative paths never have a colon on *nix nor Windows.  This moduleSpecification could be an
+    // :artifactId or groupId:artifactId for a different artifact.
+    if (moduleSpecification.contains(":")) {
```

Comment:
Hmm... this is kind of a nit, but what about drive letters on Windows? `C:\workspace\...`?

Reply:
On second thought, this is OK. We just support relative path, and that's it.

Reply:
Maven requires the paths to be a relative path:
```
 -pl,--projects <arg>                   Comma-delimited list of specified
                                        reactor projects to build instead
                                        of all projects. A project can be
                                        specified by [groupId]:artifactId
                                        or by its relative path
```